### ISSUE TITLE
Bench material

### ISSUE THREAD

**Issue by 32Vache** [streetcomplete/StreetComplete#3132]
### General
Affected tag(s) to be modified/added: [material=*](https://wiki.openstreetmap.org/wiki/Key:material) on [amenity=bench](https://wiki.openstreetmap.org/wiki/Tag:amenity=bench)
Question asked: **What is the material of this bench?**

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/wiki/Adding-new-Quests-to-StreetComplete)):
- [x] 🚧 To be added tag is established and has a useful purpose
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one)
- [x] 🐿️ Easily answerable by everyone from the outside but a survey is necessary
- [x] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of map data (Worth the effort)

### Ideas for implementation

**Element selection:** 
Bench materials. Wood, Steel, Stone, Metal, Concrete, Brick, Iron, and others

**Metadata needed:** 
None

**Proposed UI:**
Just like road/path surface, ask user what material the bench is using pictures of benches made with corresponding material.

**Comment by westnordost** [comment]
So what is the useful purpose?

**Comment by smichel17** [comment]
I did a quick edit to fix the checklist format, no other changes.

**Comment by 32Vache** [comment]
> So what is the useful purpose?

The material of benches is something that can't be mapped without surveying, it is not visible through aerial imagery or other editing sources for OSM contributors mapping @ home, similar to whether benches have a backrest. It would be quite a quick quest for the user to rapidly check the material of the bench.

**Comment by westnordost** [comment]
The fact that it may be easy to survey does not make the information itself useful.

**Comment by opk12** [comment]
- In the summer, most metal benches are unusable or PITA for most of the day, and stone and concrete make the difference.
- Wood feels easier on my back than concrete and brick. I guess medical conditions amplify the difference (chronic pain, crutches).

**Comment by 10992-osm** [comment]
> So what is the useful purpose?

Wooden benches might degrade or disappear more quickly, so if nothing else, their existence could be checked more often.

Also, if it's wet, untreated wooden benches retain moisture for longer.

Looking at taginfo (global and GB), the vast majority of benches which already have a `material=*` tag have `material=wood`.

**Comment by smichel17** [comment]
> their existence could be checked more often.

Currently they're checked every 6 years by default. So, if this is often enough for wood benches, we can use that value for all benches; if it's not enough, it should probably be lowered, since most are wood. :P

**Comment by westnordost** [comment]
Okay... this is all far fetched though IMO. Then, we'd also need to record if a bench is in the shadow/covered by a tree etc.

**Comment by mnalis** [comment]
> Okay... this is all far fetched though IMO. Then, we'd also need to record if a bench is in the shadow/covered by a tree etc.

I guess you were joking, but now that you mention it, I _do_ map trees near bench (and on correct sides of it) to indicate there might be shadow. I find that after I've cycled many dozens of kilometers at 30+ degrees Celsius, I do not want to sit on the bench under the full power of sun (this is multiplied on bicycle, because when I stop, the refreshing wind stops too, so I start heating up even more).

While detailed, such practice is not ideal for searching though (now, if OsmAnd offered custom Overpass queries, this would rock).  But this issue let me found out about https://wiki.openstreetmap.org/wiki/Key:shade, which I didn't know about, but will keep in mind. (although it is not as clearly defined to map as bench `material`).
But I digress.

So anyway this quest looks way more useful to me than for example "the material of power pole" quest (which is useful maybe only to few people repairing power poles? Who _hopefully_ have their own database of power poles anyway and do not depend on OSM for their work), and about as useful as bench beckrest quest.

**Comment by westnordost** [comment]
No whataboutism please. I we'd always argue that there is also a more or at least as unimportant quest included, then we would slowly progress over to things like `sidewalk:left:color=#656565`.
As for shade, the sun obviously moves, so its really really subjective what may be considered as in the shade at what time. A really bad idea for a tag.

**Comment by matkoniecz** [comment]
> So anyway this quest looks way more useful to me than for example "the material of power pole" quest (which is useful maybe only to few people repairing power poles? Who hopefully have their own database of power poles anyway and do not depend on OSM for their work), and about as useful as bench beckrest quest.

if I would do anything it would be disabling this quests by default rather than adding more similar ones

**Comment by mnalis** [comment]
> No whataboutism please. I we'd always argue that there is also a more or at least as unimportant quest included, then we would slowly progress over to things like `sidewalk:left:color=#656565`.

Actually, it would only happen in we always included quests that are _about as important_ (if only quest **more important** that power pole material were included, then it would never get less important quests) - but **I get your idea**. I guess quests that were included in the past (when there was a lot less quests), would not be included today. (And I do apologize to the author, whoever they might be, for picking up that particular example - I didn't mean it in negative context). 

I was just trying to indicate that there are uses for that (there are [more than 10,000](https://taginfo.openstreetmap.org/keys/bench#combinations) `material` tags on the `amenity=bench`, so obviously some people care enough), and try to explain why I personally care about that and somewhat related issue; and others have given their reasons. 

If the quest were written by someone and accepted, I'd surely enable it in my presets which has `backrest` quest. If it were not to be accepted, I'd survive - in most of my use cases where I care for benches (cycling outside of cities), benches are in most cases not mapped at all, so I have to add them (along with backrest and material, yes) from OsmAnd (or Vespucci) anyway. 

> As for shade, the sun obviously moves, so its really really subjective what may be considered as in the shade at what time. A really bad idea for a tag.

Well, if one looks at its [wiki](https://wiki.openstreetmap.org/wiki/Key:shade), it says it's for `places that are shaded in most seasons and most times of day, e.g. because they are covered by vegetation, as in the photo`, so not such a horrible idea as it looks at first glance when I hears it without more details. (And yes, I agree there are many more situations that are in _"sometimes in shade, sometimes not"_ category). 

But I was not actually proposing that SC asks for a shade (even if I think it is useful information) - primarily because I'm afraid many SC users wouldn't take full picture into consideration and thus answer such quest wrongly. For `bench material` however, I do not think there is such potential about wrong answer, so it is IMHO only a matter of _**"Is it useful enough?"**_

**Comment by mnalis** [comment]
Anyway, to put my money where my mouth is, ***IF such quest were determined to be useful enough to be accepted***, and interested parties would help gather acceptable pictures of benches in various materials, I'd volunteer for trying to implement such quest (might be a reasonably complex quest for my first try)

(Of course, @westnordost  if you're still not convinced it is useful, eg. it wouldn't be accepted, I have better things to do)

**Comment by carstenhag** [comment]
Just to chime in on the discussion, in the past I have mapped a few benches with Vespucci and added the material & backrest tags, as it was a preset. I don't think it's useful either.

**Comment by westnordost** [comment]
Alright, I go with my gut and reject this quest suggestion. If the bench material has been included in a preset, it explains the high number of it being tagged on benches.

**Comment by mnalis** [comment]
> If the bench material has been included in a preset, it explains the high number of it being tagged on benches

@westnordost I think you misunderstood what @carstenhag was saying - Vespucci **offers to you to select** what material of the bench is (the same as it does for backrest or number of seats etc), but it **does not prefill it**.

eg. creating new bench in Vespucci looks like this:
![bench_vespucci](https://user-images.githubusercontent.com/156656/129606123-758e8dbc-ca97-44c4-8a44-5ccdacf7c069.jpg)

In other words, the high number of `material=*` taggings on the benches are because high number of people has invested extra effort to click that field and select the value (eg. they cared enough to do that extra effort), and **not** because _"`material=wood` was pre-selected by default"_.

(Now, whether that number of tags is enough to convince you the quest might be useful in SC is another thing; I just wanted to clear up the **misunderstanding** above as it seemed to influenced your judgement on the issue)

**Comment by westnordost** [comment]
Yes I know, that is what I meant.
